src/routes/invite/+page.svelte

```diff
@@ -42,10 +43,22 @@
 </svelte:head>
 
 <Unauthenticated>
-    <svelte:fragment slot="title">Invite</svelte:fragment>
+    <svelte:fragment slot="title">
+        {#if !userId || !secret || !membershipId || !teamId}
+            Invalid invite
+        {:else}
+            Invite
+        {/if}
+    </svelte:fragment>
     <svelte:fragment>
         {#if !userId || !secret || !membershipId || !teamId}
-            <p class="text">Invalid invite link.</p>
+            <Alert type="warning">
+                <svelte:fragment slot="title">We couldn’t find a valid invite</svelte:fragment>
+                Please ask the project owner to provide you with a new invite to their project
```

Comment:
This kinda sounds weird, can you check with @gewenyu99 🙂

Reply:
I wouldn't use "couldn't find", sounds like we're searching and we weren't sure.

Maybe: "The invite link is not valid. Please ask the project owner to send you a new invite."

But still better than the original!